### DISCUSSION TITLE
help with french translation

### DISCUSSION THREAD

**Discussion by camille-granier** [Syknapse/Contribute-To-This-Project#1298] [Q&A] [answered]
Hi! 

I'm looking to make my first contribution to an open-source project. I noticed some typos and a few sentences that haven't been translated in the french translation. I would be very happy to help with if that's ok?

French is my mother tongue.

**Comment by Preet-Sojitra** [comment, marked as answer]
@camille-granier Heyy!! You can work on it. After translating, push your code and create pull request, it will be merged soon 👍
